### ISSUE TITLE
Error when uploading new Manifest_Users json (v5.5 to v5.5.2)

### ISSUE THREAD

**Issue by tgainey** [OfficeDev/microsoft-teams-apps-company-communicator#1424]
I am attempting to update from version 5.5 to 5.5.2,

I've successfully uploaded the Authors folder, but receive an error message when attempting to upload the Users folder.
![Capture](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/143223810/86e18a18-2f2a-4a53-8fb3-07277472b582)

**Comment by tgainey** [comment]
???

**Comment by v-jaygupta** [comment]
Hello @tgainey ,

Thanks for reaching out to us!

Could you please upload the author app manifest package ZIP file to the below Teams app validator site and see if its giving any error - 
https://dev.teams.microsoft.com/appvalidation.html

**Comment by tgainey** [comment]
Thank you!  That link helped.  I placed a new image file in the folder and it seemed to accept that.  But after updating the files, and adding the app back into my Teams permission policy, now I cannot add the app back to the Teams site?

**Comment by gsv022** [comment]
Once the zipped manifest file is ready, delete the existing manifest file in the teams admin center then add the new zipped one. It will take some time to reflect into teams. After 10-15minutes, please check it under "Built for your organization."

**Comment by tgainey** [comment]
> Once the zipped manifest file is ready, delete the existing manifest file in the teams admin center then add the new zipped one. It will take some time to reflect into teams. After 10-15minutes, please check it under "Built for your organization."

Not sure where to "delete" in TAC?  I uploaded the new .zip and still cannot add the app back to the Teams site:

![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/143223810/8bec3437-a98d-49d9-a9fa-b11eec60c3bb)

**Comment by gsv022** [comment]
In the above screenshot, right-top there will be delete option please delete it and re-upload it

**Comment by tgainey** [comment]
so I need to delete both apps entirely and re-upload?  Wouldn't I have to add again from Azure?  This seems extreme?

**Comment by gsv022** [comment]
Yes,  since you made modifications to the manifest folder, it is recommended to re-upload them again. To delete the existing manifest files,  Here is the reference.
<img width="755" alt="image" src="https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/98143492/ac595e0f-b446-47b7-b38c-35a3c6d31776">

**Comment by tgainey** [comment]
ok, I have deleted and redeployed, I can now access the app!  However, our Marketing Director cannot.  He is receiving a message that there was a problem reaching this app:

![image](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/assets/143223810/cc5f2d7a-4d4d-4765-9130-b60a8a3a9534)

He has tried from the desktop and web apps, we've even reset the desktop app/cleared cache.  He was able to send messages a few months ago before the update.  He has appropriate author permissions, and the Secret does not expire until next year.  I've restarted the web app and all function apps also.  We really needed to get this running this week, and we have an important communication to send before 5/18.
